doc/usecases/snippet_sharing/UC_duplicate_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Create configuration snippet from existing snippet

```

Comment:
I think my wording describes better what is happening. "Duplicate" sounds to me like an atomic action that does not allow editing in the same step, but my implementation will allow editing before the snippet gets actually stored (frontend-only feature). So there is never really a duplicate created (except when the user wants to do so).